examples/website-demos/virtual-event/src/assets/EventCardIcon.tsx

```diff
@@ -0,0 +1,10 @@
+export const EventCardIcon = () => {
+  return (
+    <svg width='24' height='25' viewBox='0 0 24 25' fill='none' xmlns='http://www.w3.org/2000/svg'>
+      <path
+        d='M21 1.93115H3C2.46957 1.93115 1.96086 2.14187 1.58579 2.51694C1.21071 2.89201 1 3.40072 1 3.93115C1 4.46159 1.21071 4.97029 1.58579 5.34537C1.96086 5.72044 2.46957 5.93115 3 5.93115V15.9312C3 16.7268 3.31607 17.4899 3.87868 18.0525C4.44129 18.6151 5.20435 18.9312 6 18.9312H9.92L7.24739 22.335C7.09134 22.5427 7.02226 22.803 7.05478 23.0608C7.0873 23.3185 7.21886 23.5535 7.42161 23.716C7.62436 23.8784 7.88238 23.9556 8.14104 23.9312C8.39969 23.9067 8.63868 23.7825 8.80739 23.585L12 19.5612L15.2474 23.585C15.4161 23.7825 15.6551 23.9067 15.9137 23.9312C16.1724 23.9556 16.4304 23.8784 16.6332 23.716C16.8359 23.5535 16.9675 23.3185 17 23.0608C17.0325 22.803 16.9634 22.5427 16.8074 22.335L14.08 18.9312H18C18.7956 18.9312 19.5587 18.6151 20.1213 18.0525C20.6839 17.4899 21 16.7268 21 15.9312V5.93115C21.5304 5.93115 22.0391 5.72044 22.4142 5.34537C22.7893 4.97029 23 4.46159 23 3.93115C23 3.40072 22.7893 2.89201 22.4142 2.51694C22.0391 2.14187 21.5304 1.93115 21 1.93115ZM19 15.9312C19 16.1964 18.8946 16.4507 18.7071 16.6383C18.5196 16.8258 18.2652 16.9312 18 16.9312H6C5.73478 16.9312 5.48043 16.8258 5.29289 16.6383C5.10536 16.4507 5 16.1964 5 15.9312V5.93115H19V15.9312Z'
+        fill='#8A898E'
```

Comment:
should this be a variable? also you can implicit return these components without additional logic in the component